src/ice_shelf/MOM_ice_shelf_diag_mediator.F90

```diff
@@ -344,6 +345,35 @@ subroutine post_IS_data(diag_field_id, field, diag_cs, is_static, mask)
 
 end subroutine post_IS_data
 
+!> Make a real ice shelf scalar diagnostic available for averaging or output
+subroutine post_IS_data_0d(diag_field_id, field, diag_cs, is_static)
+  integer,           intent(in) :: diag_field_id !< The id for an output variable returned by a
+                                                 !! previous call to register_diag_field.
+  real,              intent(in) :: field         !< real value being offered for output or averaging
+  type(diag_ctrl), target, intent(in) :: diag_CS !< Structure used to regulate diagnostic output
+  logical, optional, intent(in) :: is_static !< If true, this is a static field that is always offered.
+  ! Local variables
+  real :: locfield
```

Comment:
Please add a comment with the description of every real variable and their units, both here and in `register_MOM_IS_scalar_field()` below.  In this case, the units of `field` are probably `in arbitrary rescaled units [A ~> a]` while those for `locfield` are probably `in arbitrary unscaled units [a]`.  There are now a number of analogous examples in MOM_diag_mediator.F90 that could be used as a template.